### PR TITLE
Document filter parameters on machine entities

### PR DESCRIPTION
**Pull request by ricardosantacattarina**

### CODE REVIEW

**Comment by ricardosantacattarina** [comment]
[[Postman] Include endpoints with filter](https://trello.com/c/yHcdjCjP/455-postman-include-endpoints-with-filter)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -4675,6 +4675,54 @@
...
+												{
+													"key": "name",
+													"value": "Case IH  7120",
+													"description": "Filter by machine name",
+													"disabled": true
+												},
+												{
+													"key": "provider",
+													"value": "CaseIH",
+													"description": "Fitler by Provider",
```
Comment (Vinniekun) [inline Leaf API.postman_collection.json:4688]: typo
```suggestion
													"description": "Filter by Provider",
```

**Review by Vinniekun — APPROVED** [review]
Nice work

**Review by palomacione — APPROVED** [review]

File: Leaf API.postman_collection.json (hunk 1 above)
Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json:4688]: Nice catch!

Comment (ricardosantacattarina) [reply Leaf API.postman_collection.json:4688]: https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/60/commits/49ccc23f3c31575137f69642fb840d2fa71abc8b

File: .idea/Leaf-API-Postman-Collection.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (fabyan-alexander) [inline .idea/Leaf-API-Postman-Collection.iml:9]: Is this file necessary? This is new for me

**Review by fabyan-alexander — CHANGES_REQUESTED** [review]
Nice update!
Can we improve the description by explaining what kind of data is expected in each filter and avoid using "Filter by" because the endpoint is a query

File: .idea/Leaf-API-Postman-Collection.iml (hunk 2 above)
Comment (ricardosantacattarina) [reply .idea/Leaf-API-Postman-Collection.iml]: woops, that wasn't intended. https://github.com/Leaf-Agriculture/Leaf-API-Postman-Collection/pull/60/commits/36602d4203e375b2c569a6575e0b4b0d17e3e559

**Review by fabyan-alexander — APPROVED** [review]
